awx/network_ui_test/CONTRIBUTING.md

```diff
@@ -0,0 +1,221 @@
+
+Network UI Test
```

Comment:
Is this all just test code? What does it test? This whole module seems to duplicate a lot of code. It's testing but requires its own models, views, migrations, etc? I think I'd prefer a more traditional unit/functional test paradigm that doesn't require a totally different deployment to run.

Reply:
This code was moved to https://github.com/benthomasson/network_ui_test